Ports/RISC-V/RV32/GCC/os_cpu_a.S

```diff
@@ -153,19 +169,92 @@ OSStartHighRdy:
 #              the result of an interrupt.  This function triggers a synchronous software interrupt by
 #              writing into the MSIP register
 #********************************************************************************************************
+OSIntCtxSw:
+    ret
 
 OSCtxSw:
-OSIntCtxSw:
-# MIE_MSIE -- enable software interrupt bit
-    li     t0, RISCV_MIE_MSIE
-    csrrs  zero, mie, t0
-
-# This will trigger a synchronous software interrupt; PRCI->MSIP[0] = 0x01;
-    li     t0, RISCV_PRCI_BASE_ADDR
-    li     t1, 0x1
-    sw     t1, 0x0(t0)
+    ecall
     ret
 
+.align 8
```

Comment:
Put a comment saying in RISCV exception handler MUST be 8 byte aligned

Reply:
Sorry, I think align 8 aligns to 1<<8 i.e 256 byte so change the comment to say "256 bytes"